### DISCUSSION TITLE
Move code suggestion

### DISCUSSION THREAD

**Discussion by ctb0k33** [aptos-labs/aptos-developer-discussions#138] [Questions] [answered]
gm everyone, is there any extension in VS code that help code suggestion in move ? For example, i use aptos_framework::account and try to call account::create_account_for_test. But the editor not suggest this method.
![ảnh](https://github.com/aptos-labs/aptos-developer-discussions/assets/96061934/12275244-7792-4718-b9a2-162b58753293)

**Comment by 0x-j** [comment, marked as answer]
Hi the vscode move analyzer is not fully working at the moment, we recommend you use IntelliJ (free version is good enough) with move plugin made by Pontem.

**Reply by ctb0k33** [reply]
thanks for information !
